src/components/Admin/index.jsx

```diff
@@ -26,18 +28,50 @@ import { formatTimestamp } from '../../utils';
 import './Admin.scss';
 
 class Admin extends React.Component {
+  constructor(props) {
+    super(props)
+    const { location } = props;
+    const queryParams = qs.parse(location.search);
+    this.state = {
+      searchQuery: queryParams.search || '',
+      searchSubmitted: !!queryParams.search
+    }
+  }
   componentDidMount() {
     const { enterpriseId } = this.props;
     if (enterpriseId) {
       this.props.fetchDashboardAnalytics(enterpriseId);
     }
+    if (location.pathname != `${this.props.enterpriseSlug}/admin/learners`){
+      this.isSearchShow = false
```

Comment:
@iamhaiderkhan what is the usage of this variable?

Reply:
its a flag for search limit to the only main report page.

Reply:
where are you using this flag?

Reply:
`src/components/Admin/index.jsx` L313